oneview_redfish_toolkit/handler_multiple_oneview.py

```diff
@@ -24,7 +24,10 @@
 
 RESOURCE_STRATEGY = {
     "appliance_node_information": {"get_version": st.all_oneviews_resource},
-    "connection": {"get": st.first_parameter_resource},
+    "connection": {
+        "get": st.first_parameter_resource,
+        "post": st.create_server_profile,
+        },
```

Comment:
Maybe it's better to remove the mapped strategy "create_server_profile" for "server_profiles.create" from this same dict.
Once it's being changed how the strategy(create_server_profile) handles the arguments, so even that we are not using right now, if we call "g.oneview_client.server_profiles.create()" it will have an error.